src/main/java/dk/kb/ds/cumulus/export/CumulusExport.java

```diff
@@ -117,6 +124,12 @@ public static void main(String[] args) throws Exception {
         }
     }
 
+    private static String getString(String verbatim_value, String originalCumulusValue, boolean b) {
+        if (b) {
+            verbatim_value = originalCumulusValue;
+        }
+        return verbatim_value;
+    }
```

Comment:
The idea of making a method for this is fine. The name `getString` does not convey what is happening and the `b`-value is always calculated by checking if the `verbatim_value` is `null`. What about moving the `null`-check inside of the method and change the method signature to be `getStringWithFallback(String str, String fallback)`? (Not sure if `getStringWithFallback` is the best name, but something along those lines).

Reply:
I will do some renaming. The check is actually not of the verbatim value but on the calculated UTC time so I still need that value check.